docs/streams/getting-started-streams.md

```diff
@@ -109,9 +111,11 @@ async function streams() {
     try{
       await console.log("Creating local stream...");
       const stream_local = await client.createStream("testStream-local", true);
-
-      await console.log("Creating global stream...");
+      console.log('Local Stream:', stream_local.result["stream-id"]);
+      
+      await console.log("\nCreating global stream...");
```

Comment:
Why is the /n necessary? We do not have that in any of the other console.log lines I have seen.

Reply:
It's for readability since it puts the next step on a new line. Without it, there would be 4 output lines in a row vs separated based on the type of stream that was created